sklearn/impute/tests/test_impute.py

```diff
@@ -657,12 +661,12 @@ def test_iterative_imputer_imputation_order(imputation_order):
 @pytest.mark.parametrize(
     "estimator", [None, DummyRegressor(), BayesianRidge(), ARDRegression(), RidgeCV()]
 )
-def test_iterative_imputer_estimators(estimator):
-    rng = np.random.RandomState(0)
+def test_iterative_imputer_estimators(estimator, global_random_seed):
+    rng = np.random.RandomState(64)
```

Comment:
I think this should use the global seed as well (maybe a leftover from debugging?):
```suggestion
    rng = np.random.RandomState(global_random_seed)
```